Source/WebCore/platform/mediarecorder/cocoa/AudioSampleBufferCompressor.mm

```diff
@@ -326,7 +334,7 @@
     }
 
     CMSampleBufferRef rawSampleBuffer;
-    auto error = PAL::CMAudioSampleBufferCreateWithPacketDescriptions(kCFAllocatorDefault, buffer.get(), true, NULL, NULL, m_destinationFormatDescription.get(), numPackets, m_currentNativePresentationTimeStamp, m_destinationPacketDescriptions.data(), &rawSampleBuffer);
+    auto error = PAL::CMAudioSampleBufferCreateWithPacketDescriptions(kCFAllocatorDefault, buffer.get(), true, NULL, NULL, m_destinationFormatDescription.get(), numPackets, m_currentNativePresentationTimeStamp, &m_destinationPacketDescriptions, &rawSampleBuffer);
```

Comment:
This is only ever true if numPackets is 1.
Given we go in that direction, I would remove numPackets from sampleBufferWithNumPackets and rename it to just sampleBuffer or something like that.